source/tests/system/nirfsa_driver_api_tests.cpp

```diff
@@ -599,9 +599,9 @@ TEST_F(NiRFSADriverApiTests, ErrorMessage_ReturnsErrorMessage)
   const auto response = client::error_message(stub(), session, IVI_ATTRIBUTE_NOT_SUPPORTED_ERROR);
 
   EXPECT_SUCCESS(session, response);
-  EXPECT_EQ(
-      "IVI: (Hex 0xBFFA0012) Attribute or property not supported.",
-      response.error_message());
+  EXPECT_THAT(
+      response.error_message(),
+      HasSubstr("IVI: Attribute or property not supported."));
```

Comment:
This still has the `IVI: ` prefix, so it's not a `substr` of the original message. Shouldn't it be the same as the one below?